searchlib/src/vespa/searchlib/attribute/imported_attribute_vector.h

```diff
@@ -27,25 +26,25 @@ class ReferenceAttribute;
  * Any accessor on the imported attribute for a local LID yields the same result as
  * if the same accessor were invoked with the target LID on the target attribute vector.
  */
-class ImportedAttributeVector : public ReadableAttributeVector {
```

Comment:
Consider dropping the 'attribute::' prefix as ImportedAttributeVector is located in namespace search::attribute. The same goes for the rest of the header file and cpp file regarding change to ReadableAttributeVector.